e2e/tests-dfx/create.bash

```diff
@@ -352,3 +352,48 @@ teardown() {
   assert_contains 'Freezing threshold: 2_592_000'
   assert_contains 'Log visibility: controllers'
 }
+
+@test "create with multiple log allowed viewer list in dfx.json" {
```

Comment:
same 2 scenarios: users have no access & do have access.

Reply:
This test is used to cover the canister can be created with the allowed view list in dfx.json. The [create with default settings](https://github.com/dfinity/sdk/blob/a685c3f4f41a92da469ed0fa5a8e93d470c05e7b/e2e/tests-dfx/create.bash#L344) covers the no access case.